pkg/insights/metrics.go

```diff
@@ -32,7 +36,11 @@ func MustRegisterMetrics(registrables ...metrics.Registerable) {
 }
 
 func init() {
-	MustRegisterMetrics(RecommendationCollector)
+	MustRegisterMetrics(RecommendationCollector, counterRequestSend)
+}
+
+func IncrementCounterRequestSend(status string) {
```

Comment:
change signature to accept `int` instead. Http codes type is `int` and we can play with a default value of the same type. Then before sending the data, transform it with `strconv` or `fmt` as string.